### PR TITLE
Fix dev-es stale GH workflow

### PR DESCRIPTION
**Pull request by electrocucaracha**
### Describe your changes
This change fixes the syntax of stale workflow, removing the double quotes from the name and upgrades the version number of the task

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by krol3** [comment]
@electrocucaracha @ramrodo  I was testing this action, after we merge in to the main branch, the last (before)  will work fine.  I dont think we need this pr to fix it. It was your suggestion @ramrodo

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -1,28 +1,43 @@
-name: "Stale label"
+---
+# SPDX-license-identifier: Apache-2.0
```
Comment (krol3) [inline .github/workflows/stale.yml:2]: Do we need of these generated comments? my suggestion is a clean yml file

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -1,28 +1,43 @@
...
-        exempt-pr-labels: 'maintainers'
-        stale-pr-label: 'stale'
-        stale-issue-label: 'stale'
-        days-before-stale: 60
-        days-before-close: 20
-        only-labels: 'lang/es'
-        close-issue-label: 'close/stale'
-        close-pr-label: 'close/stale' # Add the label for mainteiners track the close pr labels.
\ No newline at end of file
+      - uses: actions/stale@v8.0.0
```
Comment (krol3) [inline .github/workflows/stale.yml:30]: it was tested with this new version this action?

File: .github/workflows/stale.yml (hunk 1 above)
Comment (electrocucaracha) [reply .github/workflows/stale.yml:2]: IMHO, it's a good practice to have headers on files, those can eventually help on SBOM adoption. The Spanish spellchecker contains the same header

File: .github/workflows/stale.yml (hunk 2 above)
Comment (electrocucaracha) [reply .github/workflows/stale.yml:30]: I tried to test this locally with `act`  tool but requires some env vars. Besides using the latest version released, this semver format is validated by dependabot

**Review by ramrodo — APPROVED** [review]

**Comment by seokho-son** [comment]
I guess this workflow needs to be applied to main (base) branch. :)
I think we can apply this PR to main branch first, and open another PR to update dev-es.

File: .github/workflows/stale.yml (hunk 2 above)
Comment (krol3) [reply .github/workflows/stale.yml:30]: ok!! let's merge it!

**Review by krol3 — APPROVED** [review]
LGTM

**Comment by electrocucaracha** [comment]
> I guess this workflow needs to be applied to main (base) branch. :) I think we can apply this PR to main branch first, and open another PR to update dev-es.

This file doesn't exist in main, so I was thinking to fix it first in `dev-es` and submit other PR to sync changes with `main` later.

**Review by seokho-son — APPROVED** [review]
Currently, this PR is targeting to the main branch. :)
Anyway, LGTM as is.

**Pull request merged**